client/src/main/java/com/kritsit/casetracker/client/domain/services/Administrator.java

```diff
@@ -111,8 +117,9 @@ private InputToModelParseResult validate(Map<String, Object> inputMap){
         for(Map.Entry<String, Object> entry : inputMap.entrySet()){
             if(entry.getKey().equals("firstname")) continue;
             if(entry.getKey().equals("position")) continue;
-            if(entry.getKey().equals("permission")&& entry.getValue() instanceof Permission) continue;
```

Comment:
Isn't it validated with StringValidator? because if its empty, its added to failed inputs, as shown in the tests.